src/main/kotlin/com/noumenadigital/npl/cli/commands/registry/DeployCommand.kt

```diff
@@ -62,11 +62,11 @@ class DeployCommand(
 
         val sourceDirFile = File(sourceDirValue)
         if (!sourceDirFile.exists()) {
-            output.error("Source directory does not exist: ${sourceDirFile.absolutePath}")
+            output.error("Source directory does not exist: $sourceDirValue")
```

Comment:
we just echo the user's input back to them, therefore we don't have to worry about the file separators or w/e in tests now